src/py/mat3ra/notebooks_utils/preamble/material.py

```
@@ -0,0 +1,20 @@
+from mat3ra.made.material import Material
+from mat3ra.made.tools.build.defective_structures.zero_dimensional.point_defect.point_defect_type_enum import (
+    PointDefectTypeEnum,
+)
+from mat3ra.made.tools.build.pristine_structures.zero_dimensional.nanoparticle.enums import NanoparticleShapesEnum
+from mat3ra.made.tools.build_components.entities.reusable.zero_dimensional.coordinates_shape_enum import (
+    CoordinatesShapeEnum,
+)
+from mat3ra.made.tools.helpers import *  # noqa: F403
+from mat3ra.made.tools.helpers import __all__ as _HELPER_NAMES
+
+__all__ = [
```

Ruff (0.16.1)

[error] Invalid object in `__all__`, must contain only strings

(PLE0604)